core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.html

```diff
@@ -219,7 +223,7 @@
       flex-wrap: wrap;
       width: 50%;
     }
-    .save-changes-btn {
+   .save-changes-btn {
```

Comment:
Why is this changed?

Reply:
This is not a change as such. Once space at the beginning was removed, probably it happened when I used VS Code's formatter. Do you want to restore the space?

Reply:
Fixed